common/scripts/setup_env.sh

```diff
@@ -175,17 +175,14 @@ fi
 # LOCAL_OUT should point to architecture where we are currently running versus the desired.
 # This is used when we need to run a build artifact during tests or later as part of another
 # target.
-if [[ "${FOR_BUILD_CONTAINER:-0}" -eq "1" ]]; then
-  LOCAL_OUT="${TARGET_OUT_LINUX}"
-else
-  LOCAL_OUT="${TARGET_OUT}"
-fi
-
 if [[ "${FOR_BUILD_CONTAINER:-0}" -eq "1" ]]; then
   # Override variables with container specific
   TARGET_OUT=${CONTAINER_TARGET_OUT}
   TARGET_OUT_LINUX=${CONTAINER_TARGET_OUT_LINUX}
+  LOCAL_OUT="${TARGET_OUT_LINUX}"
```

Comment:
This changes LOCAL_OUT to point to CONTAINER_TARGET_OUT_LINUX which is a location within the container instead of a location on the host. In this case we are running in a Linux container so that is correct. It will also point to a directory within the contianer.

Reply:
As I think about this more, LOCAL_OUT has and still needs to point the the LOCAL host structure, else it will have an adverse affect on the `docker` script.